end-to-end-tests-localdb/junit/results-0-0..xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@pvannierop this one should prolly be excluded?

Reply:
File removed.